### DISCUSSION TITLE
Recreational length sample sizes

### DISCUSSION THREAD

**Discussion by shcaba** [shcaba/Sebastes_melanops_OR#18] [General]
How were we going to define the effective sample size for recreational lengths?
I have the lengths formatted, I just need the sample sizes by year and month.

**Comment by aliwhitman** [comment]
As far as I know, common practice is to just use number of fish for recreational data. Would number of sampled trips be a better measure?

**Comment by shcaba** [comment]
Sampled trips would be a better one, if available.

On Tue, Mar 28, 2023 at 11:14 AM Ali Whitman ***@***.***>
wrote:

> As far as I know, common practice is to just use number of fish for
> recreational data. Would number of sampled trips be a better measure?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/shcaba/Sebastes_melanops_OR/discussions/18#discussioncomment-5457381>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB6IZLRNOJGKGHSYL5IAGGDW6MTALANCNFSM6AAAAAAWJXVOQY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 

*Jason M. Cope, Ph.D.Research Fishery Biologist*
*Fishery Resource Analysis and Monitoring Division*
*Northwest Fisheries Science Center*
*2725 Montlake Blvd. East*

*Seattle, WA 98112-2013*
*NOAA Fisheries*
*Affiliate assistant faculty*
*School of Aquatic and Fishery Sciences*
*University of Washington*
***@***.*** ***@***.***>
(206) 302-2417
www.nmfs.noaa.gov

**Comment by shcaba** [comment]
And this is for lengths. Number of fish is used for conditional age at
lengths.


On Tue, Mar 28, 2023 at 11:16 AM Jason Cope - NOAA Federal <
***@***.***> wrote:

> Sampled trips would be a better one, if available.
>
> On Tue, Mar 28, 2023 at 11:14 AM Ali Whitman ***@***.***>
> wrote:
>
>> As far as I know, common practice is to just use number of fish for
>> recreational data. Would number of sampled trips be a better measure?
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/shcaba/Sebastes_melanops_OR/discussions/18#discussioncomment-5457381>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AB6IZLRNOJGKGHSYL5IAGGDW6MTALANCNFSM6AAAAAAWJXVOQY>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***
>> com>
>>
>
>
> --
>
> *Jason M. Cope, Ph.D.Research Fishery Biologist*
> *Fishery Resource Analysis and Monitoring Division*
> *Northwest Fisheries Science Center*
> *2725 Montlake Blvd. East*
>
> *Seattle, WA 98112-2013*
> *NOAA Fisheries*
> *Affiliate assistant faculty*
> *School of Aquatic and Fishery Sciences*
> *University of Washington*
> ***@***.*** ***@***.***>
> (206) 302-2417
> www.nmfs.noaa.gov
>
>
>


-- 

*Jason M. Cope, Ph.D.Research Fishery Biologist*
*Fishery Resource Analysis and Monitoring Division*
*Northwest Fisheries Science Center*
*2725 Montlake Blvd. East*

*Seattle, WA 98112-2013*
*NOAA Fisheries*
*Affiliate assistant faculty*
*School of Aquatic and Fishery Sciences*
*University of Washington*
***@***.*** ***@***.***>
(206) 302-2417
www.nmfs.noaa.gov

**Comment by aliwhitman** [comment]
Okay, on it.  I think I could get this from RecFIN with the right query...

**Comment by aliwhitman** [comment]
Small detail question on this - are we looking for the number of trips that were sampled? Or the number of trips that were sampled that had black rockfish?  Something I should have thought to ask earlier...

**Comment by shcaba** [comment]
I would think of it as a trip number associated with every length.
The effective sample size would then be the number of unique trips within a
strata, with the strata being year and month.
So instead of number of individuals sampled per month within a year by sex
(though I think the sexes are basically unsexed, correct?), it would be
number of trips by month by year by sex.

On Wed, Mar 29, 2023 at 10:49 AM Ali Whitman ***@***.***>
wrote:

> Small detail question on this - are we looking for the number of trips
> that were sampled? Or the number of trips that were sampled that had black
> rockfish? Something I should have thought to ask earlier...
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/shcaba/Sebastes_melanops_OR/discussions/18#discussioncomment-5468875>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB6IZLVBYAZZ4HC56RIWSODW6RY2PANCNFSM6AAAAAAWJXVOQY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 

*Jason M. Cope, Ph.D.Research Fishery Biologist*
*Fishery Resource Analysis and Monitoring Division*
*Northwest Fisheries Science Center*
*2725 Montlake Blvd. East*

*Seattle, WA 98112-2013*
*NOAA Fisheries*
*Affiliate assistant faculty*
*School of Aquatic and Fishery Sciences*
*University of Washington*
***@***.*** ***@***.***>
(206) 302-2417
www.nmfs.noaa.gov

**Comment by aliwhitman** [comment]
Okay, a little hiccup with this. I have a RecFIN query that provides the number of trips associated with the BIO and CPFV black rockfish lengths. These are all unsexed (blank) but by year and month.  That was easy and can be provided. 

However, if we are also including the aged samples in the length comps (ORA samples), this is not an accurate counting of the trips associated with all the lengths, as I don't have a way to use the correct trip identifier with the aged samples in RecFIN (just a quirk of how the tables are structured).  I've got an email into Jason Edwards and our ODFW sport folks about this.  

But thoughts?
